utils/exporters/blender/addons/io_three/constants.py

```diff
@@ -329,3 +329,4 @@
 
 DOUBLE_SIDED = 'doubleSided'
 
+EXPORT_SETTINGS_KEY = 'three_export_settings'
```

Comment:
I used this format, because all addons I use add properties using the underscore-separated keys. But I am totally fine with camelCase and adjusted the PR.